chapters/si/_toctree.yml

```diff
@@ -0,0 +1,201 @@
+- title: 0. සැකසුම්
+  sections:
+    - local: chapter0/1
+      title: හැඳින්වීම
+
+- title: 1. පරිවර්තක ආකෘති
+  sections:
+    - local: chapter1/1
+      title: හැඳින්වීම
+    - local: chapter1/2
+      title: ස්වභාවික භාෂා සැකසීම
+    - local: chapter1/3
+      title: පරිවර්තක, එම පරිවර්තක වලට කුමක් කළ හැකිද?
+    - local: chapter1/4
+      title: පරිවර්තක ක්‍රියාකරන්නේ කෙසේද ?
+    - local: chapter1/5
+      title: සංකේතක ආකෘති
+    - local: chapter1/6
+      title: විකේතක ආකෘති
+    - local: chapter1/7
+      title: අනුක්‍රමිකයක සිට අනුක්‍රමිකයකට ආකෘති
+    - local: chapter1/8
+      title: නැඹුරුවීම් සහ සීමාවන්
+    - local: chapter1/9
+      title: සාරාංශය
+    - local: chapter1/10
+      title: පරිච්ඡේදය අවසානයේ ප්‍රශ්නාවලිය
+      quiz: 1
+
+- title: 2. Using 🤗 Transformers
```

Comment:
Let's remove the untranslated sections from the ToC to avoid mixing Sinhala and English chapters on the hf.co website :)

Reply:
Fixed that. Hope to continue the work. @lewtun  :)